resteasy-jaxrs/src/main/java/org/jboss/resteasy/plugins/server/servlet/HttpServletResponseWrapper.java

```diff
@@ -48,13 +48,13 @@ public void write(byte[] bytes, int i, int i1) throws IOException
       @Override
       public void flush() throws IOException
       {
-         response.getOutputStream().flush();
+         //NOOP (RESTEASY-1650)
       }
 
```

Comment:
@asoldano , SseEventOutputImpl retrieves response with ResteasyProviderFactory.getContextData(HttpServletResponse.class) to flush sse event. So this won't effect SseEventOutputImpl.writeEvent(), right ?